### PR TITLE
Extract documents into another project;

### PR DESCRIPTION
**Pull request by StanislavChankovSellerCloud**
* Extracted Documents related classes/enums;

### CODE REVIEW

File: src/SellerCloud.Core.Documents/Enums/InitialLabelRotation.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿namespace SellerCloud.Core.Enums.CarrierFeatureProviderType
+﻿namespace SellerCloud.Core.Documents.Enums
```
Comment (stanimirsellercloud) [inline src/SellerCloud.Core.Documents/Enums/InitialLabelRotation.cs]: Looks like the `InitialLabelRotation` enum has been moved out.

1. Is this going to require updating the library in Pdf Formatters?
2. Do you think we should rename the enum to `DocumentOrientation`?

File: src/SellerCloud.Core.Documents/SellerCloud.Core.Documents.csproj [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <Description>Contains SellerCloud's domainless classes/enums.</Description>
```
Comment (stanimirsellercloud) [inline src/SellerCloud.Core.Documents/SellerCloud.Core.Documents.csproj]: Let's update the `Description`. e.g. `Contains SellerCloud's document-related classes/enums.`

File: src/SellerCloud.Core.Documents/Enums/InitialLabelRotation.cs (hunk 1 above)
Comment (stanimirsellercloud) [reply src/SellerCloud.Core.Documents/Enums/InitialLabelRotation.cs]: (or `Rotation` because the enum values are degree-related)

Comment (StanislavChankovSellerCloud) [reply src/SellerCloud.Core.Documents/Enums/InitialLabelRotation.cs]: Sure, `Rotation` is okay I think.

**Pull request merged**